pdns/secpoll-auth.cc

```diff
@@ -45,7 +45,7 @@ static vector<ComboAddress> parseResolveConf()
       for(vector<string>::const_iterator iter = parts.begin()+1; iter != parts.end(); ++iter) {
 	
 	try {
-	  ret.push_back(ComboAddress(*iter, 53));
+	  s_servers.push_back(ComboAddress(*iter, 53));
```

Comment:
I kept to using the existing nomenclature for this variable, but it might help to rename it to `s_secpollresolvers`.